lib/saml_idp/name_id_formatter.rb

```diff
@@ -13,14 +13,18 @@ def all
         one_one.map { |key_val| build("1.1", key_val)[:name] } +
         two_zero.map { |key_val| build("2.0", key_val)[:name] }
       else
-        list.map { |key_val| build("2.0", key_val)[:name] }
+        list.map do |key_val|
+          format_symbol = key_val.is_a?(Array) ? key_val[0] : key_val
```

Comment:
Can simplify this:

```suggestion
          format_symbol = Array(key_val).first
```

Reply:
ooh I saw that lower down but didn't realize how that worked - I'll update.